secular_measure_of_work_ethic_survey/survey.tsv

```diff
@@ -0,0 +1,14 @@
+question_type	question_text	required	page_number	option_text	option_values
```

Comment:
This survey also has parsing/loading issues - there are commas in some of the text so they are split further. This will need further debugging.

Reply:
It sounds weird. It is a tsv file which is supposed to be separated by tab, not commas? Some other survey.tsv files also have commas in their text.

Reply:
There are a bit of both - the overall file is tab delimited, but within a field (e.g., options) the comma delineates. So for some of these questions, when you break apart by tabs then commas, the number of values != the number of options so it throws an error.

Reply:
Fixed depression-anxiety-stress-scales-21-survey. You can check it on my newest commit.

Reply:
Fixed secular-measure-of-work-ethic-survey. I think the reason of the bug is that I used . (period) instead of , (comma) in option_values. It should work properly now.

Reply:
okay we're good to go for the secular measure survey! It's added https://github.com/expfactory-experiments/secular-measure-of-work-ethic-survey. I haven't tested the depression scales yet - will do later this evening.